### PR TITLE
OCPVE-616: fix: add conditions to error on missing lv in case VG is present but thin-pool is not

### PR DESCRIPTION
**Pull request by jakobmoellerdev**
This adds more failure conditions where the LV could be corrupt and the LVMCluster will now go into degraded state.

Specifically three conditions are created:

1. If the `lvs` report of LVM is empty
2. If the `lvs` report of LVM is not empty but has no part for any LV
3. If the `lvs` report of LVM is not empty but only contains entries for LVs that are not the thin-pool we created.

For all three cases, previously the LVMCluster would remain in "Ready", now it will be "Degraded" instead

### CODE REVIEW

File: pkg/vgmanager/vgmanager_controller.go [hunk 1]
```diff
@@ -382,12 +382,22 @@ func (r *VGReconciler) validateLVs(volumeGroup *lvmv1alpha1.LVMVolumeGroup) erro
 	if err != nil {
 		return fmt.Errorf("could not get logical volumes found inside volume group, volume group content is degraded or corrupt: %w", err)
 	}
+	if len(resp.Report) < 1 {
+		return fmt.Errorf("LV report was empty, meaning that the thin-pool LV is no longer found, " +
```
[the remark this answers is not in the document]
Comment (jakobmoellerdev) [reply pkg/vgmanager/vgmanager_controller.go:386]: I don't understand why its bad to have concatenation? We never used "`" in the entire Repo and only have 3 cases of errors.New()? Could you explain why thats better?

[the remark this answers is not in the document]
Comment (brandisher) [reply pkg/vgmanager/vgmanager_controller.go:386]: Nits inherently don't block merging.

That said, I'm likely to blame for setting a precedent on this type of suggestion. I'm generally quite against concatentation as there's rarely a good case to use it. Happy to explain further but that's probably best done outside of a PR :smile:

**Pull request merged**